tools/cldr-apps/src/main/java/org/unicode/cldr/web/UserRegistry.java

```diff
@@ -186,8 +184,10 @@ private synchronized void notify(User u) {
 
     /**
      * List of all user levels - for UI presentation
-     **/
-    public static final int ALL_LEVELS[] = { ADMIN, TC, MANAGER, EXPERT, VETTER, STREET, LOCKED };
+     *
+     * Code related to UserRegistry.EXPERT removed 2021-05-18 per CLDR-14597
+     */
+    public static final int ALL_LEVELS[] = { ADMIN, TC, MANAGER, VETTER, STREET, LOCKED };
 
```

Comment:
probably a separate PR (or even ticket), but this should ideally be generated from VoteResolver.Level

Reply:
i'll file a separate issue